OWNERS_ALIASES

```diff
@@ -132,6 +132,8 @@ aliases:
     - deads2k
     - stevekuznetsov
     - wojtek-t
+  wg-structured-logging-leads:
+    - serathius
```

Comment:
Would be nice if we ensure we have at least two leads.

Reply:
> Would be nice if we ensure we have at least two leads.

Agree. I don't want to be a blocker to getting this WG started and am good with this as a follow-up but we usually ask SIG/WG/UGs to have at least two leads to avoid single points of failure.

Reply:
Done